server.lua

```diff
@@ -69,7 +69,14 @@ ServerCallback.Register('openInventory', function(source, inv, data)
 					right = Inventory(owner and stash.name..owner or stash.name)
 
 					if not right then
-						right = Inventory.Create(owner and stash.name..owner or stash.name, stash.label or stash.name, inv, stash.slots, 0, stash.weight, owner or false)
+						right = Inventory.Create(owner and stash.name..owner or stash.name, stash.label or stash.name, inv, stash.slots, 0, stash.weight, owner or false, nil, stash.degrade)
+					end
+					if stash.degrade == false then
+						for a,b in pairs(right.items) do
+							if b.metadata.refrigerate then
```

Comment:
I feel like checking for `refrigerate` item metadata is maybe a bit too specific naming?

Reply:
Yup, right. I've changed it to degradeamount